ros_ws/src/extractor_context.txt

```diff
@@ -0,0 +1,7 @@
+You are an expert in grammatical structures and will be given a sentence from which 
```

Comment:
I've been using `.prompt` for text files. No one ever uses `.txt`; if you don't like `.prompt`, then use no extension which is more "normal".